falcon/statics.py

```diff
@@ -0,0 +1,31 @@
+import os.path
+import falcon
+
+
+class Statics:
+
+    def __init__(self, prefix, parent_folder):
+        self.prefix = prefix
+        self.parent_folder = os.path.abspath(parent_folder)
+
+
+    def _get_stream(self, path):
+        with open(path, 'rb') as f:
+            return f
+
+    def __call__(self, req, resp):
+        stripped = req.path[len(self.prefix):]
+        normalized = os.path.normpath(stripped).lstrip('/')
+
+        if normalized.startswith('../'):
```

Comment:
Also ensure it does not start with `'/'`. Assuming `self.prefix` ends in a forward slash, there should be no reason why the normalized path should start with one. And if it does (and there should be a test case for this), you end up jumping out of `self.parent_folder`:

```bash
In [8]: os.path.join('/usr/share/nginx/www', '/etc')
Out[8]: '/etc'

```